src/Storage/Storage/Blob/StorageDataMovementCmdletBase.cs

```diff
@@ -112,20 +112,7 @@ protected override void BeginProcessing()
 
         protected virtual void DoBeginProcessing()
         {
-            CmdletOperationContext.Init();
-            CmdletCancellationToken = _cancellationTokenSource.Token;
-            WriteDebugLog(String.Format(Resources.InitOperationContextLog, GetType().Name, CmdletOperationContext.ClientRequestId));
-
-            if (_enableMultiThread)
-            {
-                SetUpMultiThreadEnvironment();
-            }
-
-            OpContext.GlobalSendingRequest +=
-                (sender, args) =>
-                {
-                    //https://github.com/Azure/azure-storage-net/issues/658
-                };
+            base.BeginProcessing();
```

Comment:
The removed steps are already in it's base's BeginProcessing(), so remove them.
https://github.com/Azure/azure-powershell/blob/e8b923485b270d19e2a55f68b27a738caf324e6a/src/Storage/Storage/Common/StorageCloudCmdletBase.cs#L619